src/x11_in_gtk3.c

```diff
@@ -303,6 +313,11 @@ suil_x11_wrapper_idle(void* data)
 
 	wrap->idle_iface->idle(wrap->instance->handle);
 
+	if (wrap->is_resized) {
+		gtk_widget_set_size_request(GTK_WIDGET(wrap), wrap->min_width, wrap->min_height);
```

Comment:
Those didn't resize the widget, it just remove the previous set of size request to allow the user to shrink the window below the initial size. It "can't" lead to a resize of the window, because you cant shrink the window below this values. gtk_widget_set_size_request only resize the window if the new size_request is above the current size.

In the x11_in_gtk wrapper XGetNormalHints is already in use, and they allow to set min-size, max-size, aspect ratio as well, why do you think XSizeHints be to prefer?

Reply:
PS. `suil_x11_wrapper_idle` is only used if there is a `LV2_UI__idleInterface` extension.

Reply:
> gtk_widget_set_size_request only resize the window if the new size_request is above the
> current size

It also, imposes a new minimum size.

Reply:
If a X11 plugin UI ain't have set the min_size hint, this will set to (-1,-1) which means native free.
If a plugin has set min_size, this will used here.

Could you drive a X11 UI without LV2_UI__idleInterface?